.github/workflows/publish.yml

```diff
@@ -0,0 +1,100 @@
+name: Publish
+
+on:
+  # This allows us to run this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
+      version-strategy:
+        type: choice
+        description: 'Version strategy'
+        required: true
+        options:
+            - 'patch'
+            - 'minor'
+            - 'major'
+        default: patch
+      title:
+        type: string
+        description: 'Release title'
+        required: true
+
+concurrency:
+  group: deploy
+  cancel-in-progress: true
+
+# We need these permissions create new GitHub releases
+permissions: write-all
+
+jobs:
+  update_package_version:
+    name: Update package version
+    runs-on: ubuntu-latest
+    steps:
+      - name: Notify Slack about new release
+        uses: ravsamhq/notify-slack-action@v2
+        with:
+          notification_title: "⏳ Starting new constructor-ui-autocomplete deployment"
+          status: ${{ job.status }}
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+      - uses: actions/checkout@v4
+      - uses: oleksiyrudenko/gha-git-credentials@v2-latest
+        with:
+          token: '${{ secrets.GITHUB_TOKEN }}'
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '18.12.x'
+      - name: Install dependencies
+        run: npm ci
+      - name: Bump release version
+        run: npm version ${{ github.event.inputs.version-strategy }}
```

Comment:
Should this be something like `npm version ${{ github.event.inputs.version-strategy }} -m ${{ github.event.inputs.title }}`?

Reply:
You're right. I added this with the latest commit